example/image-classification/symbols/symbol_inception-v4.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding:utf-8 -*-
```

Comment:
yeah, we trained using this network. but not reproduce the numbers, not for val_acc.
this network is copy from paper entirely.